src/query/graphite/native/builtin_functions.go

```diff
@@ -242,6 +243,42 @@ func timeShift(
 	}, nil
 }
 
+
+func timeSlice(ctx *common.Context, inputPath singlePathSpec, start string, end string) (ts.SeriesList, error) {
```

Comment:
nit: Can you add a description for this function, maybe something similar to the docs for the function?